docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,210 @@
+
+You can create a provider as follows:
+
+- Open the Yugabyte Platform web UI and click **Configure a Provider** to open the **Cloud Provider Configuration** page shown in the following illustration.
+- Select **Red Hat OpenShift** and complete the fields, as follows:
+  - In the **Type** filed, select **OpenShift**.
+  - In the **Name** field, enter ocp-test.
+  - Use the **Kube Config** field to select the file that you created in the preceding step.
+  - In the **Service Account** field, enter yugabyte-platform-universe-management.
+  - In the **Image Registry** field, if you are performing Operator-based installation, use  `registry.connect.redhat.com/yugabytedb/yugabyte`, and if you are performing Helm-based installation, use  `quay.io/yugabyte/yugabyte-ubi`
+- Optionally, use the **Pull Secret File** field to upload the pull secret you received from Yugabyte Support. 
+
+![OpenShift Provider Config](/images/ee/openshift-cloud-provider-setup.png)
+
+- Click **Add Region** and complete the **Add new region** dialog shown in the following illustration by first selecting the region you found previously (US East), and then entering the following information:
+  - In the **Zone** field, enter the exact zone label (us-east4-a).
+  - In the **Namespace** field, enter yb-platform.
+
+![img](https://lh5.googleusercontent.com/gQ-fTRZnDTp4kCqNaKo8KUgVA2mJeCCjaHiqzdCOcG4350yxgDGZojMWhsfdvcpLJbOio8sL8K932wmDM6_S8fIkL9wfKAo-b3340Yvc1Dy4FJ61o2Ec93DptkB_Ski1XqaU3UrT)
+
+- Click **Add Region**.
+- Click **Save**. 
+
+You should see the newly-added provider under **Red Hat OpenShift configs**.
+
+## Creating a Universe Using the Provider
+
+You can create a universe using the provider as follows:
+
+- Use the Yugabyte Platform web UI to navigate to **Universes** and then click **Create Universe**. 
+
+- Complete the **Create Universe** page shown in the following illustration by entering the following information:
+
+  - In the **Name** field, enter universe-1.
+  - In the **Provider** field, enter ocp-test.
+  - In the **Regions** field, enter US East.
+  - In the **Instance Type** field, enter xsmall (2 cores, 4GB RAM).
+
+![img](https://lh3.googleusercontent.com/E8EwHd0olx_ZmmpRN-vpem8TAvqqU_gPQGyuMJH9yDxKS89cswTURFASkAE0fDKWMCfImUknxbyN7z87SzaOJJE_PO6Wo4aKw7UHIdXy7oCxPgTmv2XvRMvAIvbRQygS0gz3oXNz)
+
+- Click **Create**. 
+
+The following illustration shows the universe creation progress:
+
+![img](https://lh3.googleusercontent.com/wmVq3lUnqseriSgXcw-gwqNRct4CxUSZ0NnGwgVJ5UG-7GU8Ja4kKHGUZ428BmYOZvcemMLlod_9PBWQAFaj99bOybyW_XEc40X6mZL9fZKi6IQ9q7-Cr52XTfzjmw0ppGTN8b9Z)
+
+If you click universe-1 and then navigate to **Tasks**, you should see a page similar to the one shown in the following illustration:
+
+![img](https://lh3.googleusercontent.com/UFeWUHRm7Amxek1U8yxqeV3z4RiUll05Vo9VowiCImlELaCabHyaDFFT81XYbbBXUMoDAK8g13q9FP4ZbwOSAoCzsqGV83zqF9Jkcw_3pb4U4WxgC7MOiULbCpAArxIqSHMrDDIC)
```

Comment:
ocp-yugabyte-platform-universe-3.png

Reply:
I, actually, saved all the images on the drive already and replaced them in the doc.